src/index.html

```diff
@@ -7,12 +7,62 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="item"
+      data-qa="card"
+    >
+      <header class="item__header">
+        <img
+          src="./images/imac.jpeg"
+          alt="item"
+        />
+      </header>
+      <main class="item__main">
+        <div class="item__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </div>
+        <div class="item__code">Product code: 195434</div>
+        <div class="item__reviews">
+          <div class="item__reviews_rating stars">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <div class="item__reviews_count">Reviews: 5</div>
+        </div>
+        <div class="item__price">
+          <div class="item__price_text">Price:</div>
+          <div class="item__price_number">$2,199</div>
+        </div>
+      </main>
+      <footer class="item__footer">
```

Comment:
```suggestion
```
wrong usage of the footer tag (same as header)
but wrapper is redundant here - just add styles you need to the button